profile.md

```diff
@@ -421,3 +421,12 @@ specific behaviour.
 | Broker API Field | Type | Description |
 | --- | --- | --- |
 | metadata.shareable | string | Allows Service Instances to be shared across orgs and spaces. |
+| metadata.dashboard_client | [DashboardClient](#dashboard-client-object) | Contains the data necessary to activate the Dashboard SSO feature for this service. |
```

Comment:
I think we should take a step back here - this doesn't seem like a great fit for metadata or the catalog endpoint in general, as-is.

In kubernetes, we take whatever's in metadata and put it into a user-visible field. AFAIK we don't have an expectation that the platform name is sent to the catalog endpoint. That means that when a broker returns this field, it will be visible to every user that can see the service resource.  That means that if you return this information, you've opened it up to a wider audience than you probably expected.

I don't wish to stall this at all, but I think we need to begin reasoning about the catalog endpoint as returning information that may be visible to everyone, even users with a low degree of privilege. Could we consider moving this to a separate resource or subresource in the future?

Reply:
So in this regard, CF and k8s are behaving differently. In CF, the platform uses information in `metadata` to understand behaviours but doesn't make this information visible to the end user. We also only show some information from the catalog in general, hence this kind of information has been stored here.

Do you think this is probably a good input for the v3 design?

Reply:
In general it does seem like we may need to have a clean separation between data that's ok for end users to see and data that is not ok for end users to see - and each needs to be clearly identified in the spec so that Platforms can act appropriately.  In this case, if we were to move this data into some "secretMetadata" section then Kube can know that it should do something special with it.

Reply:
Actually CF and K8s are the same here. We expose all information returned in metadata on the CF API. Historically metadata has been populated with data for client tooling. This would be the first time we have to "hide" something from the response.

If we're considering a different abstraction for broker-to-platform metadata. Do we want to move it to the current metadata object just to have to move it again later?